infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -225,17 +225,17 @@ class LocationTrackController(
     @PreAuthorize(AUTH_VIEW_DRAFT_OR_OFFICIAL_BY_PUBLISH_TYPE)
     @GetMapping("/track-numbers/{$PUBLISH_TYPE}/{trackNumberId}/location-tracks")
     fun getTrackNumberTracksByName(
-        @PathVariable("$PUBLISH_TYPE") publicationState: PublishType,
+        @PathVariable("$PUBLISH_TYPE") publishType: PublishType,
```

Comment:
Lintteri muuten nillittää näitä "$PUBLISH_TYPE" koska tuo string templating on turha: tuo PUBLISH_TYPE on jo string -> voi muuttaa vain muotoon `@PathVariable(PUBLISH_TYPE)`